README.md

```diff
@@ -52,13 +53,13 @@ Clone the project from GitHub:
 
 If you are on Linux, execute:
 
-    $ cmake -DTHIRDPARTY=ON ..
+    $ cmake ../ -DTHIRDPARTY=ON -DEPROSIMA_BUILD=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=install
```

Comment:
If you want to install Fast-RTPS in the system without examples or testing, all these options aren't needed. Telling the user to call `cmake ..` should add another command before like `mkdir build && cd build`.

Reply:
:+1: